javascript/code_to_gast/js_router.py

```diff
@@ -75,6 +75,10 @@ def node_to_gast(node):
         return js_helpers.return_statement_to_gast(node)
     elif node.type == "AssignmentPattern":
         return js_helpers.assign_pattern_to_gast(node)
+    elif node.type == "ArrowFunctionExpression":
```

Comment:
What is the reasoning behind both arrowfunctionexpression and functionexpression doing the same thing?

Reply:
The main difference between these two functions is how the keyword "this" is bound. I talked to Michael and he said for our functionality its fine to treat them the same. The arrow function is a newer feature which is why I prioritized it for py -> js